src/Preferences.js

```diff
@@ -65,6 +65,15 @@ define(function (require, exports, module) {
                 description: Strings.PREF_INDENT_DESC
             },
             inContextMenu: true
+        }, {
+            id: "autohide",
+            type: "boolean",
+            value: true,
+            options: {
+                name: Strings.PREF_AUTOHIDE_NAME,
+                description: Strings.PREF_AUTOHIDE_DESC
+            },
+            inContextMenu: true
```

Comment:
Or with its own button?

If you think autohide is an interesting feature and has merge possibility, I have no problem trying to fix it.